framework/components/ema-status/ema-status.twig

```diff
@@ -0,0 +1,51 @@
+{#
```

Comment:
This seems to be a copy/paste from another template (things like `copyright`. or `hotline` doesn't make sense here)
We don't really rely on twig template, so either you update it or you just remove irrelevant information